### ISSUE TITLE
pad clearance

### ISSUE THREAD

**Issue by ferdymercury** [pointhi/kicad-footprint-generator#545]
Is there a way to define the pad clearance with your library?

I see solder_mask_margin and solder_paste_margin, but I can not find the pad_margin?

Thanks in advance.

**Comment by ferdymercury** [comment]
also, zone_connect is missing

**Comment by evanshultz** [comment]
I don't believe settings those pad parameters is supposed. You can see at https://github.com/pointhi/kicad-footprint-generator/pull/150 I added some missing pad parameters, and if you want to submit a PR to add more perhaps that older PR will help you.

**Comment by ferdymercury** [comment]
https://github.com/pointhi/kicad-footprint-generator/pull/546
